newsfragments/2412.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+| Remove leftover imports of Twisted Logger, using instead our shim.
+| Change NuCypher Logger behavior to always escape curly braces. The rationale is that it's unclear how to justify the previous behavior with some edge cases like ``{{}``, which was processed to ``{}``.
```

Comment:
This like more than one newsfragment to me, smashed into one.  Personally, I'd prefer a second fragment with a reference to #2404

Reply:
> The rationale is that it's unclear how to justify the previous behavior with some edge cases like ``{{}``, which was processed to ``{}``.

Additionally, the voice and tone used here do not read like a news announcment from the "town crier" if you will - Instead this reads more like a PR description, or details that can be let for the ticket.

Reply:
I don't know what voice and tone is expected, but I agree that there are so details that could be left for the PR description. Updated fragment ✅